.gitignore

```diff
@@ -8,3 +8,6 @@ node_modules
 .sw?
 *.jar
 *.xpi
+.idea/
+.vscode
```

Comment:
The files produced by your choice of editor / IDE should be excluded for all Git repositories on your machine so. [Here's how to do that](https://docs.github.com/en/free-pro-team@latest/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer).

Reply:
Okay, Thank you for making me aware of that. I actually did not know if that was possible😅. I will remove them from .gitignore